### PR TITLE
Feedback @SPritchard86

### PR DESCRIPTION
**Pull request by Davvott**
@SPritchard86
Be kind! I probably haven't full PEP8 formatting

### CODE REVIEW

File: prac_05/word_occurences.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from string import punctuation
+# Prompt user for string
+text_string = input("Enter a sentence or paragraph: ")
```
Comment (SPritchard86) [inline prac_05/word_occurences.py:3]: Could force string to be lower case for more accurate sorting (I didn't do this either).

File: prac_05/convert_lists_to_dict.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+def convert_lists_to_dict(list_1, list_2):
+    converted = list(zip(list_1, list_2))  # list of tuples
+    converted = dict(zip(list_1, list_2))
+
+    temp = {}
+    for i, item in enumerate(list_1):  # Does not deal with different sized lists
+        temp[list_1[i]] = list_2[i]
+    return temp
+
+print(convert_lists_to_dict(names, dates_of_birth))
```
Comment (SPritchard86) [inline prac_05/convert_lists_to_dict.py:13]: May want to add a main() function.

File: prac_05/electricity_bill_estimator_v2.py [hunk 3]
```diff
@@ -10,18 +10,29 @@
...
+        print("Please enter an integer")
+        continue
+    if tariff not in TARIFFS:
+        print("Tariff not found, try again")
+    else:
+        correct_input = True
 
 daily_use = float(input("Enter daily use in kWh: "))
 billing_days = int(input("Enter number of billing days: "))
 total_bill = (tariff * daily_use) * billing_days
```
Comment (SPritchard86) [inline prac_05/electricity_bill_estimator_v2.py:36]: Currently total_bill is using the user input 'tariff' to multiply, not pulling the tariff from the dictionary.
try replacing tariff with: TARIFFS[tariff] or TARIFFS.get(tariff)

**Review by SPritchard86 — COMMENTED** [review]
Thanks for my feedback - the list comprehension was a good idea!

Yours is looking good! A couple things I spotted just to be nit-picky :p

word_occurences: 
Could force string to be lower case for more accurate sorting (I didn't do this either).

convert_lists_to_dict: 
May want to add a main function.

electricity_bill_estimator_v2: 
Currently total_bill is using the user input 'tariff' to multiply, not pulling the tariff from the dictionary.
try replacing tariff with: TARIFFS[tariff] or TARIFFS.get(tariff)

File: prac_05/electricity_bill_estimator_v2.py (hunk 3 above)
Comment (Davvott) [reply prac_05/electricity_bill_estimator_v2.py:36]: Thanks Steve! 
Yep I caught the tarriff error over the weekend, when my bill was massive; interesting example of a logic error that won't necessarily get picked up without testing I guess. As for the main() refactor I'm never sure what the autograder wants so I've been trying to just leave my solutions as close to the prac files as possible. Cheers mate, happy coding.

**Pull request merged**